deploy/compose/docker-compose.yml

```diff
@@ -137,6 +158,9 @@ services:
 
   grafana:
     image: "${GRAFANA_IMAGE}"
+    platform: linux/amd64
```

Comment:
Please, make this a single variable in the env file, so that it can be set up for different platforms. Or omit it completley.

Reply:
But this would make it a backwards compatible change, whereas having it inside the docker-compose file is not.